tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -532,3 +552,1116 @@ Feature: an user gets the resources shared to them
+                                "type": "object",
+                                "properties": {
+                                  "user": {
+                                    "type": "object",
+                                    "properties": {
+                                      "displayName": {
+                                        "type": "string"
+                                      },
+                                      "id": {
+                                        "type": "string"
+                                      }
+                                    },
+                                    "required": [
+                                      "displayName",
+                                      "id"
+                                    ]
+                                  }
+                                },
+                                "required": [
+                                  "user"
+                                ]
+                              }
+                            },
+                            "required": [
+                              "invitedBy"
+                            ]
+                          },
+                          "@libre.graph.permissions.actions": {
+                            "type": "array",
+                            "items": [
+                              {
+                                "type": "string"
+                              }
+                            ]
+                          },
+                          "roles": {
+                            "type": "array",
+                            "items": [
+                              {
+                                "type": "string"
+                              }
+                            ]
+                          }
+                        }
+                      }
+                    ]
+                  },
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    """
+
+  
+  Scenario: user lists the folder shared with them from project space after the sharer is deleted
+    Given using spaces DAV path
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has created a folder "folder" in space "new-space"
+    And user "Alice" has sent the following share invitation:
+      | resource        | folder    |
+      | space           | new-space |
+      | sharee          | Brian     |
+      | shareType       | user      |
+      | permissionsRole | Viewer    |
+    And the administrator has deleted user "Alice" using the provisioning API
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    """
+    {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
```

Comment:
we have the share even the sharer is deleted?

CC @SagarGi

Reply:
no we should not have the response,  i have created an issue for it lets wait if its the actual bug> https://github.com/owncloud/ocis/issues/8356

Reply:
I see that it's correct behavior
personal space: sharer is deleted - no share anymore
project space: sharer is deleted - share is exist. Another user manager can manage this share